src/entity/presentationDefinition/presentationDefinitionWrapperEntity.ts

```diff
@@ -15,7 +15,7 @@ export class PresentationDefinitionWrapperEntity {
 
   @Column()
   // @ts-ignore
-  presentation_definition: PresentationDefinition;
+  presentation_definition: PresentationDefinitionV1 | PresentationDefinitionV2;
```

Comment:
Suggest to update dependency to @sphereon/pex 0.6.3-unstable.1 as that contains an `IPresentationDefinition` type which is essentially PresentationDefinitionV1 | PresentationDefinitionV2

Reply:
Fixed!